### ISSUE TITLE
Problem with oasis cli comands

### ISSUE THREAD

**Issue by antony-everstake** [oasisprotocol/oasis-sdk#1059]
I need to known balance in emerald paratime network.When I use oasis cli I have an error.

```
./oasis accounts show oasis1qz8u0zqhcmgxalnjgwj6m0sg6wuy2vrsgyna2z7t --network mainnet --paratime emerald
Address: oasis1qz8u0zqhcmgxalnjgwj6m0sg6wuy2vrsgyna2z7t
Nonce: 60

=== CONSENSUS LAYER (mainnet) ===
  Total: 3.213897824 ROSE
  Available: 3.213897824 ROSE


  Commission Schedule:
    Rates:
      (1) start: epoch 0
          rate:  5.0%
      (2) start: epoch 2525
          rate:  15.0%
    Rate Bounds:
      (1) start:        epoch 0
          minimum rate: 0.0%
          maximum rate: 20.0%

  Stake Accumulator:
    Claims:
      - Name: registry.RegisterEntity
        Staking Thresholds:
          - Global: entity
      - Name: registry.RegisterNode.+CS8pbR8HUix6kGpUQjCv/QxMLVp9J8MkFuV+w6yRbM=
        Staking Thresholds:
          - Global: node-validator
      - Name: registry.RegisterNode./aYsC3pCFZFDak53TauD6VhMVyiHG3y8MQwNNcRUWDI=
        Staking Thresholds:
          - Global: node-compute
        Staking Thresholds:
          - Constant: 5000000000000000

Error: rpc error: code = Unavailable desc = unexpected HTTP status code received from server: 502 (Bad Gateway); transport: received unexpected content-type "text/html; charset=UTF-8"
```

When I run this command on the testnet, everything goes as it should.
```
./oasis accounts show oasis1qz7rce6dmnh9qtr9nltsyy69d69j3a95rqm3jmxw --network testnet --paratime emerald
Address: oasis1qz7rce6dmnh9qtr9nltsyy69d69j3a95rqm3jmxw
Nonce: 26

=== CONSENSUS LAYER (testnet) ===
  Total: 331.223396256 TEST
  Available: 331.223396256 TEST


  Commission Schedule:
    Rates:
      (1) start: epoch 17126
          rate:  10.0%
    Rate Bounds:
      (1) start:        epoch 17126
          minimum rate: 0.0%
          maximum rate: 25.0%

  Stake Accumulator:
    Claims:
      - Name: registry.RegisterEntity
        Staking Thresholds:
          - Global: entity
      - Name: registry.RegisterNode.NfSSIP66pWgjtzqAFEUmsjX25OD8p2gcIJ8ooBwV1ss=
        Staking Thresholds:
          - Global: node-validator
      - Name: registry.RegisterNode.a1PJ5bbUwLxLoJOwLWukJanTx/t4jVTUr8RRor/8Yjk=
        Staking Thresholds:
          - Global: node-compute


=== emerald PARATIME ===
Balances for all denominations:
  22.165067567 TEST

```

**Comment by Yawning** [comment]
A tip of tree build of the cli binary appears to work for me.

```
serenity :: Development/oasis-sdk/cli ‹main› % git rev-parse HEAD                                                                 
0985d6b71a66182bfd8fba95e4233db3ca66f5e5
serenity :: Development/oasis-sdk/cli ‹main› % ./cli accounts show oasis1qz8u0zqhcmgxalnjgwj6m0sg6wuy2vrsgyna2z7t --network mainnet --paratime emerald  

Address: oasis1qz8u0zqhcmgxalnjgwj6m0sg6wuy2vrsgyna2z7t
Nonce: 60

=== CONSENSUS LAYER (mainnet) ===
  Total: 3.213897901 ROSE
  Available: 3.213897901 ROSE


  Commission Schedule:
    Rates:
      (1) start: epoch 0
          rate:  5.0%
      (2) start: epoch 2525
          rate:  15.0%
    Rate Bounds:
      (1) start:        epoch 0
          minimum rate: 0.0%
          maximum rate: 20.0%

  Stake Accumulator:
    Claims:
      - Name: registry.RegisterEntity
        Staking Thresholds:
          - Global: entity
      - Name: registry.RegisterNode.+CS8pbR8HUix6kGpUQjCv/QxMLVp9J8MkFuV+w6yRbM=
        Staking Thresholds:
          - Global: node-validator
      - Name: registry.RegisterNode./aYsC3pCFZFDak53TauD6VhMVyiHG3y8MQwNNcRUWDI=
        Staking Thresholds:
          - Global: node-compute
        Staking Thresholds:
          - Constant: 5000000000000000


=== emerald PARATIME ===
Balances for all denominations:
  82.637923567682065579 ROSE
```

**Comment by antony-everstake** [comment]
There was a crash on the grpc oasis server. The Oasis team has fixed this issue.
